src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -1391,58 +1394,179 @@ static final Color hexToColor(String value) {
+            Map.entry("mistyrose", new Color(255, 228, 225, 255)),
+            Map.entry("moccasin", new Color(255, 228, 181, 255)),
+            Map.entry("navajowhite", new Color(255, 222, 173, 255)),
+            Map.entry("navy", new Color(0, 0, 128, 255)),
+            Map.entry("oldlace", new Color(253, 245, 230, 255)),
+            Map.entry("olive", new Color(128, 128, 0, 255)),
+            Map.entry("olivedrab", new Color(107, 142, 35, 255)),
+            Map.entry("orange", new Color(255, 165, 0, 255)),
+            Map.entry("orangered", new Color(255, 69, 0, 255)),
+            Map.entry("orchid", new Color(218, 112, 214, 255)),
+            Map.entry("palegoldenrod", new Color(238, 232, 170, 255)),
+            Map.entry("palegreen", new Color(152, 251, 152, 255)),
+            Map.entry("paleturquoise", new Color(175, 238, 238, 255)),
+            Map.entry("palevioletred", new Color(219, 112, 147, 255)),
+            Map.entry("papayawhip", new Color(255, 239, 213, 255)),
+            Map.entry("peachpuff", new Color(255, 218, 185, 255)),
+            Map.entry("peru", new Color(205, 133, 63, 255)),
+            Map.entry("pink", new Color(255, 192, 203, 255)),
+            Map.entry("plum", new Color(221, 160, 221, 255)),
+            Map.entry("powderblue", new Color(176, 224, 230, 255)),
+            Map.entry("purple", new Color(128, 0, 128, 255)),
+            Map.entry("rebeccapurple", new Color(102, 51, 153, 255)),
+            Map.entry("red", new Color(255, 0, 0, 255)),
+            Map.entry("rosybrown", new Color(188, 143, 143, 255)),
+            Map.entry("royalblue", new Color(65, 105, 225, 255)),
+            Map.entry("saddlebrown", new Color(139, 69, 19, 255)),
+            Map.entry("salmon", new Color(250, 128, 114, 255)),
+            Map.entry("sandybrown", new Color(244, 164, 96, 255)),
+            Map.entry("seagreen", new Color(46, 139, 87, 255)),
+            Map.entry("seashell", new Color(255, 245, 238, 255)),
+            Map.entry("sienna", new Color(160, 82, 45, 255)),
+            Map.entry("silver", new Color(192, 192, 192, 255)),
+            Map.entry("skyblue", new Color(135, 206, 235, 255)),
+            Map.entry("slateblue", new Color(106, 90, 205, 255)),
+            Map.entry("slategray", new Color(112, 128, 144, 255)),
+            Map.entry("slategrey", new Color(112, 128, 144, 255)),
+            Map.entry("snow", new Color(255, 250, 250, 255)),
+            Map.entry("springgreen", new Color(0, 255, 127, 255)),
+            Map.entry("steelblue", new Color(70, 130, 180, 255)),
+            Map.entry("tan", new Color(210, 180, 140, 255)),
+            Map.entry("teal", new Color(0, 128, 128, 255)),
+            Map.entry("thistle", new Color(216, 191, 216, 255)),
+            Map.entry("tomato", new Color(255, 99, 71, 255)),
+            Map.entry("transparent", new Color(0, 0, 0, 0)),
+            Map.entry("turquoise", new Color(64, 224, 208, 255)),
```

Comment:
So all of these have the extra 255 alpha arg just so this ONE entry can be different ? Do you expect others like this ? If not why not special case that in the caller method ?

Reply:
I will correct this.